src/etc/inc/filter.inc

```diff
@@ -2313,7 +2313,9 @@ function filter_generate_user_rule_arr($rule) {
 	$line = filter_generate_user_rule($rule);
 	$ret['rule'] = $line;
 	$ret['interface'] = $rule['interface'];
-	if ($rule['descr'] != "" and $line != "") {
+	if ($rule['tracker'] != "" and $line != "") {
+		$ret['descr'] = "label \"" . fix_rule_label("USER_RULE: {$rule['tracker']}") . "\"";
```

Comment:
Including the tracker id in rule description is the only way I found to track it back to count hits. what do you think if I concat description after tracker id? This way it can get counters on gui and still get human readable on /tmp/rules.debug.